docs/Developers-Guide.md

```diff
@@ -153,76 +170,29 @@ opinions, foster discussion, and move toward consensus. When responding to a pro
 
  * -1 - *Blocking no* (note: in general and when appropriate, this requires the blocker to propose something else that he/she would help put the time into doing)
 
-A proposal does *not* need to be a formal or lengthy document; it can and should be a straightforward recommendation of 
-what you want to do, ideally with a brief explanation for why it's a good idea. 
+A proposal does *not* need to be a formal or lengthy document; it can and should be a 
+straightforward recommendation of what you want to do, ideally with a brief explanation for why it's
+a good idea. 
 
-Proposals are just messages sent to the list and can be as simple as *"I think we should do X because of Y and Z. 
-Deadline for response is 2015-10-29. Assuming I've heard no blocking votes by then, I'll go ahead."*
-Note that you should make sure to include a **deadline** by which you will go ahead and do what you're proposing 
-if you don't hear any blocking responses. In general, you should leave at least 72 hours for people to respond. 
-This is not a hard-and-fast rule and you should use your best judgement in determining how far in the future the 
-deadline should be depending on the magnitude of the proposal and how much it will affect the overall project and the 
-rest of the community.
+Proposals are just messages sent to the list and can be as simple as *"I think we should do X 
```

Comment:
We don't really use this process anymore. We should probably update this to talk about the mailing list, issues, pull requests and reviews.

Reply:
I added a TODO here, feel free to update the section.